test/cli-validator/tests/expected-output.test.js

```diff
@@ -139,22 +139,23 @@ describe('cli tool - test expected output - Swagger 2', function() {
     //   example output would be [ 'Line', ':', '59' ]
 
     // errors
-    expect(capturedText[4].match(/\S+/g)[2]).toEqual('31');
-    expect(capturedText[8].match(/\S+/g)[2]).toEqual('54');
-    expect(capturedText[12].match(/\S+/g)[2]).toEqual('59');
-    expect(capturedText[16].match(/\S+/g)[2]).toEqual('108');
-    expect(capturedText[20].match(/\S+/g)[2]).toEqual('172');
+    expect(capturedText[4].match(/\S+/g)[2]).toEqual('36');
+    expect(capturedText[8].match(/\S+/g)[2]).toEqual('59');
+    expect(capturedText[12].match(/\S+/g)[2]).toEqual('31');
+    expect(capturedText[16].match(/\S+/g)[2]).toEqual('54');
+    expect(capturedText[20].match(/\S+/g)[2]).toEqual('108');
+    expect(capturedText[24].match(/\S+/g)[2]).toEqual('172');
 
     // warnings
-    expect(capturedText[25].match(/\S+/g)[2]).toEqual('36');
-    expect(capturedText[29].match(/\S+/g)[2]).toEqual('59');
-    expect(capturedText[33].match(/\S+/g)[2]).toEqual('15');
+    expect(capturedText[29].match(/\S+/g)[2]).toEqual('36');
+    expect(capturedText[33].match(/\S+/g)[2]).toEqual('59');
     expect(capturedText[37].match(/\S+/g)[2]).toEqual('15');
-    expect(capturedText[41].match(/\S+/g)[2]).toEqual('197');
-    expect(capturedText[45].match(/\S+/g)[2]).toEqual('108');
-    expect(capturedText[49].match(/\S+/g)[2]).toEqual('131');
-    expect(capturedText[53].match(/\S+/g)[2]).toEqual('134');
-    expect(capturedText[57].match(/\S+/g)[2]).toEqual('126');
+    expect(capturedText[41].match(/\S+/g)[2]).toEqual('15');
+    expect(capturedText[45].match(/\S+/g)[2]).toEqual('197');
+    expect(capturedText[49].match(/\S+/g)[2]).toEqual('108');
+    expect(capturedText[53].match(/\S+/g)[2]).toEqual('131');
+    expect(capturedText[57].match(/\S+/g)[2]).toEqual('134');
+    expect(capturedText[61].match(/\S+/g)[2]).toEqual('126');
```

Comment:
I hate these assertions since they are so fragile.  I think I tried 2 or 3 times to make them more robust but was never happy with the result.  If you have ideas about how to do this better I think we'll all welcome them.